From ea14f14d81943f80847dc073edd5811591d6d106 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Fri, 10 Mar 2023 10:02:35 +0000
Subject: [PATCH] ARM: dts: Standardise on the upstream LED names

Historically, downstream Pi DTS files have named the LEDs led0 and (if
present) led1, while upstream have used the more meaningful "ACT" and
"PWR". As part of a closer alignment with upstream, make the LED names
match.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/bcm2708-rpi-b-plus.dts   | 3 +--
 arch/arm/boot/dts/bcm2708-rpi-b-rev1.dts   | 1 -
 arch/arm/boot/dts/bcm2708-rpi-b.dts        | 1 -
 arch/arm/boot/dts/bcm2708-rpi-cm.dtsi      | 1 -
 arch/arm/boot/dts/bcm2708-rpi-zero-w.dts   | 1 -
 arch/arm/boot/dts/bcm2708-rpi-zero.dts     | 1 -
 arch/arm/boot/dts/bcm2709-rpi-2-b.dts      | 3 +--
 arch/arm/boot/dts/bcm2709-rpi-cm2.dts      | 1 -
 arch/arm/boot/dts/bcm2710-rpi-2-b.dts      | 3 +--
 arch/arm/boot/dts/bcm2710-rpi-3-b-plus.dts | 3 +--
 arch/arm/boot/dts/bcm2710-rpi-3-b.dts      | 3 +--
 arch/arm/boot/dts/bcm2710-rpi-cm3.dts      | 1 -
 arch/arm/boot/dts/bcm2710-rpi-zero-2-w.dts | 1 -
 arch/arm/boot/dts/bcm2711-rpi-4-b.dts      | 2 --
 arch/arm/boot/dts/bcm2711-rpi-400.dts      | 2 --
 arch/arm/boot/dts/bcm2711-rpi-cm4.dts      | 2 --
 arch/arm/boot/dts/bcm2711-rpi-cm4s.dts     | 1 -
 17 files changed, 5 insertions(+), 25 deletions(-)

diff --git a/arch/arm/boot/dts/bcm2708-rpi-b-plus.dts b/arch/arm/boot/dts/bcm2708-rpi-b-plus.dts
index ebfd7b4092cd..9839960c753f 100644
--- a/arch/arm/boot/dts/bcm2708-rpi-b-plus.dts
+++ b/arch/arm/boot/dts/bcm2708-rpi-b-plus.dts
@@ -157,14 +157,13 @@ &i2s {
 
 &leds {
 	act_led: led-act {
-		label = "led0";
 		default-state = "off";
 		linux,default-trigger = "mmc0";
 		gpios = <&gpio 47 0>;
 	};
 
 	pwr_led: led-pwr {
-		label = "led1";
+		label = "PWR";
 		default-state = "off";
 		linux,default-trigger = "input";
 		gpios = <&gpio 35 0>;
diff --git a/arch/arm/boot/dts/bcm2708-rpi-b-rev1.dts b/arch/arm/boot/dts/bcm2708-rpi-b-rev1.dts
index 75cde9a76016..3335f6a0f0b9 100644
--- a/arch/arm/boot/dts/bcm2708-rpi-b-rev1.dts
+++ b/arch/arm/boot/dts/bcm2708-rpi-b-rev1.dts
@@ -175,7 +175,6 @@ &i2s {
 
 &leds {
 	act_led: led-act {
-		label = "led0";
 		default-state = "off";
 		linux,default-trigger = "mmc0";
 		gpios = <&gpio 16 1>;
diff --git a/arch/arm/boot/dts/bcm2708-rpi-b.dts b/arch/arm/boot/dts/bcm2708-rpi-b.dts
index 4a0d98b530b8..634f0f6e6b49 100644
--- a/arch/arm/boot/dts/bcm2708-rpi-b.dts
+++ b/arch/arm/boot/dts/bcm2708-rpi-b.dts
@@ -157,7 +157,6 @@ &i2s {
 
 &leds {
 	act_led: led-act {
-		label = "led0";
 		default-state = "off";
 		linux,default-trigger = "mmc0";
 		gpios = <&gpio 16 1>;
diff --git a/arch/arm/boot/dts/bcm2708-rpi-cm.dtsi b/arch/arm/boot/dts/bcm2708-rpi-cm.dtsi
index de4ded34d686..ec4b154e4014 100644
--- a/arch/arm/boot/dts/bcm2708-rpi-cm.dtsi
+++ b/arch/arm/boot/dts/bcm2708-rpi-cm.dtsi
@@ -3,7 +3,6 @@
 
 &leds {
 	act_led: led-act {
-		label = "led0";
 		default-state = "off";
 		linux,default-trigger = "mmc0";
 		gpios = <&gpio 47 0>;
diff --git a/arch/arm/boot/dts/bcm2708-rpi-zero-w.dts b/arch/arm/boot/dts/bcm2708-rpi-zero-w.dts
index 6c239a4ecaf9..8ce57a10611a 100644
--- a/arch/arm/boot/dts/bcm2708-rpi-zero-w.dts
+++ b/arch/arm/boot/dts/bcm2708-rpi-zero-w.dts
@@ -213,7 +213,6 @@ &i2s {
 
 &leds {
 	act_led: led-act {
-		label = "led0";
 		default-state = "off";
 		linux,default-trigger = "actpwr";
 		gpios = <&gpio 47 GPIO_ACTIVE_LOW>;
diff --git a/arch/arm/boot/dts/bcm2708-rpi-zero.dts b/arch/arm/boot/dts/bcm2708-rpi-zero.dts
index f217bfebb3b7..7c6bbe9ca55b 100644
--- a/arch/arm/boot/dts/bcm2708-rpi-zero.dts
+++ b/arch/arm/boot/dts/bcm2708-rpi-zero.dts
@@ -154,7 +154,6 @@ &i2s {
 
 &leds {
 	act_led: led-act {
-		label = "led0";
 		default-state = "off";
 		linux,default-trigger = "actpwr";
 		gpios = <&gpio 47 GPIO_ACTIVE_LOW>;
diff --git a/arch/arm/boot/dts/bcm2709-rpi-2-b.dts b/arch/arm/boot/dts/bcm2709-rpi-2-b.dts
index 232c206bbbfb..79792d2d57ad 100644
--- a/arch/arm/boot/dts/bcm2709-rpi-2-b.dts
+++ b/arch/arm/boot/dts/bcm2709-rpi-2-b.dts
@@ -157,14 +157,13 @@ &i2s {
 
 &leds {
 	act_led: led-act {
-		label = "led0";
 		default-state = "off";
 		linux,default-trigger = "mmc0";
 		gpios = <&gpio 47 0>;
 	};
 
 	pwr_led: led-pwr {
-		label = "led1";
+		label = "PWR";
 		default-state = "off";
 		linux,default-trigger = "input";
 		gpios = <&gpio 35 0>;
diff --git a/arch/arm/boot/dts/bcm2709-rpi-cm2.dts b/arch/arm/boot/dts/bcm2709-rpi-cm2.dts
index a9e966dc0399..4e52977d3808 100644
--- a/arch/arm/boot/dts/bcm2709-rpi-cm2.dts
+++ b/arch/arm/boot/dts/bcm2709-rpi-cm2.dts
@@ -191,7 +191,6 @@ &i2s {
 
 &leds {
 	act_led: led-act {
-		label = "led0";
 		default-state = "off";
 		linux,default-trigger = "mmc0";
 		gpios = <&virtgpio 0 0>;
diff --git a/arch/arm/boot/dts/bcm2710-rpi-2-b.dts b/arch/arm/boot/dts/bcm2710-rpi-2-b.dts
index 11f12c6562cd..5bb4a9dcfb9f 100644
--- a/arch/arm/boot/dts/bcm2710-rpi-2-b.dts
+++ b/arch/arm/boot/dts/bcm2710-rpi-2-b.dts
@@ -157,14 +157,13 @@ &i2s {
 
 &leds {
 	act_led: led-act {
-		label = "led0";
 		default-state = "off";
 		linux,default-trigger = "mmc0";
 		gpios = <&gpio 47 0>;
 	};
 
 	pwr_led: led-pwr {
-		label = "led1";
+		label = "PWR";
 		default-state = "off";
 		linux,default-trigger = "input";
 		gpios = <&gpio 35 0>;
diff --git a/arch/arm/boot/dts/bcm2710-rpi-3-b-plus.dts b/arch/arm/boot/dts/bcm2710-rpi-3-b-plus.dts
index e0f0be339992..2bbb2afd8eb8 100644
--- a/arch/arm/boot/dts/bcm2710-rpi-3-b-plus.dts
+++ b/arch/arm/boot/dts/bcm2710-rpi-3-b-plus.dts
@@ -233,14 +233,13 @@ &i2s {
 
 &leds {
 	act_led: led-act {
-		label = "led0";
 		default-state = "off";
 		linux,default-trigger = "mmc0";
 		gpios = <&gpio 29 0>;
 	};
 
 	pwr_led: led-pwr {
-		label = "led1";
+		label = "PWR";
 		default-state = "off";
 		linux,default-trigger = "default-on";
 		gpios = <&expgpio 2 GPIO_ACTIVE_LOW>;
diff --git a/arch/arm/boot/dts/bcm2710-rpi-3-b.dts b/arch/arm/boot/dts/bcm2710-rpi-3-b.dts
index da6069828dde..423f1f652d29 100644
--- a/arch/arm/boot/dts/bcm2710-rpi-3-b.dts
+++ b/arch/arm/boot/dts/bcm2710-rpi-3-b.dts
@@ -248,14 +248,13 @@ &i2s {
 
 &leds {
 	act_led: led-act {
-		label = "led0";
 		default-state = "off";
 		linux,default-trigger = "mmc0";
 		gpios = <&virtgpio 0 0>;
 	};
 
 	pwr_led: led-pwr {
-		label = "led1";
+		label = "PWR";
 		default-state = "off";
 		linux,default-trigger = "input";
 		gpios = <&expgpio 7 0>;
diff --git a/arch/arm/boot/dts/bcm2710-rpi-cm3.dts b/arch/arm/boot/dts/bcm2710-rpi-cm3.dts
index c8bc0a8f24c2..ee8503040794 100644
--- a/arch/arm/boot/dts/bcm2710-rpi-cm3.dts
+++ b/arch/arm/boot/dts/bcm2710-rpi-cm3.dts
@@ -190,7 +190,6 @@ &i2s {
 
 &leds {
 	act_led: led-act {
-		label = "led0";
 		default-state = "off";
 		linux,default-trigger = "mmc0";
 		gpios = <&virtgpio 0 0>;
diff --git a/arch/arm/boot/dts/bcm2710-rpi-zero-2-w.dts b/arch/arm/boot/dts/bcm2710-rpi-zero-2-w.dts
index 8dc7fa4b4b82..fef3a60b97d5 100644
--- a/arch/arm/boot/dts/bcm2710-rpi-zero-2-w.dts
+++ b/arch/arm/boot/dts/bcm2710-rpi-zero-2-w.dts
@@ -226,7 +226,6 @@ &i2s {
 
 &leds {
 	act_led: led-act {
-		label = "led0";
 		default-state = "off";
 		linux,default-trigger = "actpwr";
 		gpios = <&gpio 29 GPIO_ACTIVE_LOW>;
diff --git a/arch/arm/boot/dts/bcm2711-rpi-4-b.dts b/arch/arm/boot/dts/bcm2711-rpi-4-b.dts
index 28c4a41c1c86..3afc315077b1 100644
--- a/arch/arm/boot/dts/bcm2711-rpi-4-b.dts
+++ b/arch/arm/boot/dts/bcm2711-rpi-4-b.dts
@@ -506,14 +506,12 @@ audio_pins: audio_pins {
 
 &leds {
 	act_led: led-act {
-		label = "led0";
 		default-state = "off";
 		linux,default-trigger = "mmc0";
 		gpios = <&gpio 42 GPIO_ACTIVE_HIGH>;
 	};
 
 	pwr_led: led-pwr {
-		label = "led1";
 		default-state = "off";
 		linux,default-trigger = "default-on";
 		gpios = <&expgpio 2 GPIO_ACTIVE_LOW>;
diff --git a/arch/arm/boot/dts/bcm2711-rpi-400.dts b/arch/arm/boot/dts/bcm2711-rpi-400.dts
index 9c49e01748a9..c54fb00b2920 100644
--- a/arch/arm/boot/dts/bcm2711-rpi-400.dts
+++ b/arch/arm/boot/dts/bcm2711-rpi-400.dts
@@ -513,14 +513,12 @@ audio_pins: audio_pins {
 
 &leds {
 	act_led: led-act {
-		label = "led0";
 		default-state = "off";
 		linux,default-trigger = "default-on";
 		gpios = <&gpio 42 GPIO_ACTIVE_HIGH>;
 	};
 
 	pwr_led: led-pwr {
-		label = "led1";
 		default-state = "off";
 		linux,default-trigger = "default-on";
 		gpios = <&expgpio 2 GPIO_ACTIVE_LOW>;
diff --git a/arch/arm/boot/dts/bcm2711-rpi-cm4.dts b/arch/arm/boot/dts/bcm2711-rpi-cm4.dts
index 0ef6af181214..ece298cf2e30 100644
--- a/arch/arm/boot/dts/bcm2711-rpi-cm4.dts
+++ b/arch/arm/boot/dts/bcm2711-rpi-cm4.dts
@@ -518,14 +518,12 @@ audio_pins: audio_pins {
 
 &leds {
 	act_led: led-act {
-		label = "led0";
 		default-state = "off";
 		linux,default-trigger = "mmc0";
 		gpios = <&gpio 42 GPIO_ACTIVE_HIGH>;
 	};
 
 	pwr_led: led-pwr {
-		label = "led1";
 		default-state = "off";
 		linux,default-trigger = "default-on";
 		gpios = <&expgpio 2 GPIO_ACTIVE_LOW>;
diff --git a/arch/arm/boot/dts/bcm2711-rpi-cm4s.dts b/arch/arm/boot/dts/bcm2711-rpi-cm4s.dts
index 81e6cc67cffe..fc9e1e423a0d 100644
--- a/arch/arm/boot/dts/bcm2711-rpi-cm4s.dts
+++ b/arch/arm/boot/dts/bcm2711-rpi-cm4s.dts
@@ -388,7 +388,6 @@ &ddc1 {
 
 &leds {
 	act_led: led-act {
-		label = "led0";
 		default-state = "off";
 		linux,default-trigger = "mmc0";
 	};
-- 
2.35.5

